applications/pelion_client/configuration/nrf52840dk_nrf52840/app_ZRelease.conf

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+
+CONFIG_PELION_CLIENT_POWER_MANAGER_ENABLE=y
+CONFIG_PELION_CLIENT_POWER_MANAGER_STAY_ON=y
+
+CONFIG_PELION_CLIENT=y
+CONFIG_PELION_NRF_SECURITY=y
+CONFIG_PELION_UPDATE_DEVELOPER=y
+
+CONFIG_PELION_CLIENT_FACTORY_RESET_REQUEST_ENABLE=y
```

Comment:
This could be slightly rearranged to:
``` suggestion
CONFIG_PELION_CLIENT=y
CONFIG_PELION_CLIENT_POWER_MANAGER_ENABLE=y
CONFIG_PELION_CLIENT_POWER_MANAGER_STAY_ON=y
CONFIG_PELION_CLIENT_FACTORY_RESET_REQUEST_ENABLE=y

CONFIG_PELION_NRF_SECURITY=y
CONFIG_PELION_UPDATE_DEVELOPER=y
```

Reply:
applies to all 52/53 configurations.

Reply:
even if the orders is alphabetical it is not correct... this is due to the fact that these options were migrated at some point.

these belong to the module
CONFIG_PELION_CLIENT
CONFIG_PELION_UPDATE_DEVELOPER

this one to the ncs
CONFIG_PELION_NRF_SECURITY

and these to the application
CONFIG_PELION_CLIENT_POWER_MANAGER_ENABLE
CONFIG_PELION_CLIENT_POWER_MANAGER_STAY_ON
CONFIG_PELION_CLIENT_FACTORY_RESET_REQUEST_ENABLE